docs/lang/articles/reference.md

```diff
@@ -198,6 +198,17 @@ compile time.
 
 ### Conditional expressions
 
+```
+conditional_expression ::=  or_test ["if" or_test "else" expression]
+expression             ::=  conditional_expression | lambda_expr
+```
+
+Conditional expressions (sometimes called a “ternary operator”) have the lowest priority of all Python operations.
+
+The expression `x if C else y` first evaluates the condition, `C` rather than `x`.
+If `C` is `true`, `x` is evaluated and its value is returned; otherwise,`y` is evaluated and its value is returned.
```

Comment:
We can refer to the `boolean expression` subsection for definition of true/false.